GEOSldas_App/ldas_setup

```diff
@@ -1118,15 +1194,15 @@ class LDASsetup:
                     print (line.rstrip().replace('GEOSldas_expid',self.rqdExeInp['EXP_ID']))
 
             # take  HISTdet or HISTens if ladas 
-            if shortfile =='HISTdet.rc' and self.ladas_coupling == 1 :
+            if shortfile =='HISTdet.rc' and self.ladas_cpl == 1 :
```

Comment:
@saraqzhang, @weiyuan-jiang: I'm trying to understand the logic of the code here.  It looks like ldas_setup first turns the default GEOSldas_HIST.rc into HISTORY.rc, then, if there is a HISTRC_FILE specified, the HISTRC_FILE overwrites the previously generated HISTORY.rc, then, if self.ladas_cpl==1, ldas_setup does the same with GEOSldas_HISTdet.rc (thereby overwriting the previously generated HISTORY.rc), and then, if self.ladas_cpl==2, ldas_setup does the same with GEOSldas_HISTens.rc (thereby overwriting the previously generated HISTORY.rc).  My sense is that the logic of this could be improved.  Also, I'm wondering if this allows for the user to specify their own HISTORY.rc in the case of self.ladas_cpl>0.  This probably needs further investigation.

Reply:
the idea ( logic)  is that  for ladas case the user is not allowed to specify his own HISTORY.rc, at least at the setup process.  After the whole thing is setup, of course the user can go to the run directory and modify the HISTORY.rc installed there if he wanted , assuming he knows not to comment out the essential collection such as incr.

Reply:
with the current development for ladas setup, nothing is specified including HISTC_FILE in the exeinp generated by ldas_setup sample,  the result will always be HISTdet.rc or HISTdens,rc  into HISTORY.rc in run/.